gatling-http/src/main/scala/io/gatling/http/Predef.scala

```diff
@@ -15,8 +15,32 @@
  */
 package io.gatling.http
 
+import io.gatling.http.check.HttpCheck
+import io.gatling.http.response.Response
+import io.gatling.http.check.HttpCheckScope
+
+import io.gatling.commons.validation.Success
+
+import io.gatling.core.session._
+import io.gatling.core.check.Check
+import io.gatling.core.check.ConditionalCheck._
+
 object Predef extends HttpDsl {
 
   type Request = org.asynchttpclient.Request
   type Response = io.gatling.http.response.Response
+
+  implicit object HttpCheckWrapper extends CheckWrapper[Response, HttpCheck] {
+    def wrap(check: Check[Response]): HttpCheck = new HttpCheck(check, HttpCheckScope.Body, None)
+  }
+
+  val HTTP_CONTENT_TYPE_HEADER_KEY: String = "Content-Type"
+  val JSON_CONTENT_TYPE_VALUE: String = "application/json"
+  
+  def isJsonResponse(response: Response): Boolean = {
+    response.isReceived && response.header(HTTP_CONTENT_TYPE_HEADER_KEY).exists { x => x.contains(JSON_CONTENT_TYPE_VALUE) }
+  }
+
+  def securedJsonCheck(check: Check[Response]): HttpCheck = checkIf((response: Response, session: Session) => Success(isJsonResponse(response)))(check)
```

Comment:
Not sure about this. This is not a standard usage: YOU need this because your web framework is flawed and your API is not properly RESTful. We can have this in the documentation though.